### DISCUSSION TITLE
NIS 2

### DISCUSSION THREAD

**Discussion by p4pr8n** [dbarzin/deming#81] [General]
Bonjour Didier,

Dans une vidéo de présentation il a été évoqué une prochaine mise à disposition de mesures pour NIS 2.

Pour ma part je participe à un groupe de travail qui vise à un renforcement général de la sécurité sur base de documents fournis par le centre pour la cybersacurité belgique avec référence à d'autres normes comme la 27000 et proposant 3 niveaux (basic, important et essentiel).

[https://atwork.safeonweb.be/fr/tools-resources/cyberfundamentals-framework](url)

Est-ce que une mise à disposition est toujours d'actualité ?
Si oui est-ce que l'idée est bien celle de partir vers ce type de mesures ?

Merci,

Philip

**Comment by dbarzin** [comment]
Bonjour Philip,

Oui, c'est toujours d'actualité. Deming permet d'importer des mesures provenant de tout référentiel. Ceux de SafeOnWeb en font bien évidement partie.

Je suis très intéressé par vos travaux, tu peux me contacter directement pour en parler.

Cordialement,
Didier
